releases/axelard/2025-11-v1.3.2.md

```diff
@@ -238,9 +238,21 @@ EOF
 
 ### Upgrade Process
 
+**Expected Migration Time:**
+- **Archival nodes:** ~12.5 hours migration + catch-up time. Peak memory: ~64GB.
+- **Validator/RPC nodes:** ~1 hour migration + catch-up time. Peak memory: ~64GB.
```

Comment:
```suggestion
- **Archival nodes:** ~12.5 hours migration + catch-up time
- **Validator/RPC nodes:** ~1 hour migration + catch-up time
```
this makes it sound like it used 64GB memory during the migration, which is then concerning if RAM is 64 GB